### ISSUE TITLE
Batch mode

### ISSUE THREAD

**Issue by PawelTroka** [philoushka/dehungarian#9]
It could be nice to have batch mode.
Something like - apply this code fix to all.

**Comment by Jaben** [comment]
This is already supported.

![image](https://cloud.githubusercontent.com/assets/448726/25488064/08cd57a8-2b34-11e7-92de-6070f0d806a5.png)

**Comment by PawelTroka** [comment]
Great, are you planing a new release soon?

**Comment by Jaben** [comment]
Yes -- I'd like to see a new release, as well. But, the VS extension has to be updated by @philoushka .
